assets/xml/objects/object_bba.xml

```diff
@@ -1,56 +1,82 @@
+        <DList Name="gBbaNeckDL" Offset="0x3760" />
+        <DList Name="gBbaHeadDL" Offset="0x3968" />
+        <DList Name="gBbaMiddleLegsDL" Offset="0x3F68" />
+        <DList Name="gBbaLowerLegsDL" Offset="0x4078" />
+        <DList Name="gBbaFeetDL" Offset="0x4180" />
+
+        <!-- Animations -->
+        <Animation Name="gBbaWalkingAnim" Offset="0x4910" /> <!-- Looks to be unused -->
+        <Animation Name="gBbaKnockedOverAnim" Offset="0x5154" />
+        <Animation Name="gBbaLyingDownAnim" Offset="0x58B8" />
+        <Animation Name="gBbaIdleHoldingBagAnim" Offset="0x5DC4" />
+
+        <!-- Skeleton -->
+        <Limb Name="gBbaUpperLegsLimb" Type="Standard" EnumName="BBA_LIMB_UPPER_LEGS" Offset="0x5DE0" />
+        <Limb Name="gBbaMiddleLegsLimb" Type="Standard" EnumName="BBA_LIMB_MIDDLE_LEGS" Offset="0x5DEC" />
+        <Limb Name="gBbaLowerLegsLimb" Type="Standard" EnumName="BBA_LIMB_LOWER_LEGS" Offset="0x5DF8" />
+        <Limb Name="gBbaFeetLimb" Type="Standard" EnumName="BBA_LIMB_FEET" Offset="0x5E04" />
+        <Limb Name="gBbaUpperRootLimb" Type="Standard" EnumName="BBA_LIMB_UPPER_ROOT" Offset="0x5E10" />
+        <Limb Name="gBbaNeckLimb" Type="Standard" EnumName="BBA_LIMB_NECK" Offset="0x5E1C" />
+        <Limb Name="gBbaHeadLimb" Type="Standard" EnumName="BBA_LIMB_HEAD" Offset="0x5E28" />
+        <Limb Name="gBbaBagLimb" Type="Standard" EnumName="BBA_LIMB_BAG" Offset="0x5E34" />
+        <Limb Name="gBbaTorsoLimb" Type="Standard" EnumName="BBA_LIMB_TORSO" Offset="0x5E40" />
+        <Limb Name="gBbaLeftUpperArmLimb" Type="Standard" EnumName="BBA_LIMB_LEFT_UPPER_ARM" Offset="0x5E4C" />
+        <Limb Name="gBbaLeftLowerArmRootLimb" Type="Standard" EnumName="BBA_LIMB_LEFT_LOWER_ARM_ROOT" Offset="0x5E58" />
+        <Limb Name="gBbaLeftForearmLimb" Type="Standard" EnumName="BBA_LIMB_LEFT_FOREARM" Offset="0x5E64" />
+        <Limb Name="gBbaLeftHandLimb" Type="Standard" EnumName="BBA_LIMB_LEFT_HAND" Offset="0x5E70" />
+        <Limb Name="gBbaRightUpperArmLimb" Type="Standard" EnumName="BBA_LIMB_RIGHT_UPPER_ARM" Offset="0x5E7C" />
+        <Limb Name="gBbaRightLowerArmRootLimb" Type="Standard" EnumName="BBA_LIMB_RIGHT_LOWER_ARM_ROOT" Offset="0x5E88" />
+        <Limb Name="gBbaRightForearmLimb" Type="Standard" EnumName="BBA_LIMB_RIGHT_FOREARM" Offset="0x5E94" />
+        <Limb Name="gBbaRightHandLimb" Type="Standard" EnumName="BBA_LIMB_RIGHT_HAND" Offset="0x5EA0" />
+        <Skeleton Name="gBbaSkel" Type="Flex" LimbType="Standard" LimbNone="BBA_LIMB_NONE" LimbMax="BBA_LIMB_MAX" EnumName="BbaLimbs" Offset="0x5EF0" />
+
+         <!-- Animations -->
+        <Animation Name="gBbaIdleAnim" Offset="0x6550" />
+        <Animation Name="gBbaWalkingHoldingBagAnim" Offset="0x6B10" />
+
+        <!-- TLUT -->
+        <Texture Name="gBbaTLUT" OutName="bba_tlut" Format="rgba16" Width="16" Height="16" Offset="0x6B20" />
+
+        <!-- Unused Textures -->
+        <!-- These look to just be exact copies of the used textures-->
```

Comment:
You could probably test this in a hex editor if you want to be sure.

Reply:
Confirmed with diffing the `inc.c`. They are the exact same.